src/pages/index.js

```diff
@@ -112,20 +102,26 @@ const Home = ({ data, location }) => {
         <ResponsiveSpacer>
           <FeatureBlock
             imageSide="right"
-            backgroundImageUrl={`url(${undrawBuildingWebsites})`}
+            imageSrc={securityMaintenanceIllustration}
             text={
               <div>
-                <h2 className={classes.h2}>Custom plugins</h2>
-                <p className={classes.p}>Ready to roll, easy to extend.</p>
+                <h2 className={classes.h2}>Maintenance and Security built in</h2>
+                <p className={classes.p}>Backstage moves quickly and it takes effort to stay up to date.</p>
                 <p className={classes.p}>
-                  Common SaaS tools come baked in. When you need to break out you can roll your own
-                  and deploy with a simple push.
+                  Enter some details about your custom plugin into Roadie, copy some credentials, then `npm publish` into our private and secure pipeline. Slack notifications included.
```

Comment:
This line seems copy paste from line 78

Reply:
Yep. I was just copy/pasting text to fill out the visuals. It's fixed up now.